### PR TITLE
[MULTIARCH-3807] Set checkmarks for disconnected install

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.15 +

Issue: 
- https://issues.redhat.com/browse/MULTIARCH-3807

Link to docs preview: https://71869--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/preparing-to-install-on-ibm-z

QE review:
- [x] QE has approved this change.

Additional information: Disconnected installations have also been tested now see comment here: 
https://github.com/openshift/openshift-docs/pull/69850#issuecomment-1947914257

### CODE REVIEW

**Comment by SNiemann15** [comment]
/label peer-review-needed

QE approval pending

**Comment by holgwolf** [comment]
LGTM

File: installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc [hunk 1]
```diff
@@ -27,9 +27,8 @@ The {product-title} installation program offers the following methods for deploy
...
 
-* *Local Agent-based*: You can deploy a cluster locally with the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]. It provides many of the benefits of the {ai-full}, but you must download and configure the Agent-based Installer first. Configuration is done with a command line interface (CLI).
+* *Local Agent-based*: You can deploy a cluster locally with the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]. It provides many of the benefits of the {ai-full}, but you must download and configure the Agent-based Installer first. Configuration is done with a command line interface (CLI). This approach is ideal for air-gapped or restricted networks.
```
Comment (xenolinux) [inline installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc:30]: https://redhat-documentation.github.io/supplementary-style-guide/#disconnected 
I did see "air-gapped" being used for bare metal and VSphere. So feel free to ignore my comment if "air-gapped" suits best here.

**Review by xenolinux — COMMENTED** [review]
A minor comment.But mostly LGTM!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc (hunk 1 above)
Comment (SNiemann15) [reply installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc:30]: Thanks for sharing I didn't know that but I like the term "disconnected" most. Will change

Comment (SNiemann15) [reply installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc:30]: @mjpytlak @bscott-rh Is that something we should look into? I think in most of the install docs we use restricted.

Comment (bscott-rh) [reply installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc:30]: There has been discussion over time about whether "restricted" or "disconnected" is the preferred term for this. I'm not aware of any real conclusion from those discussions.

File: installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc [hunk 2]
```diff
@@ -46,7 +45,7 @@ The {product-title} installation program offers the following methods for deploy
 
 |Restricted network {ibm-z-name} with z/VM
 |
-|
+|&#10003;
 |xref:../../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[&#10003;]
```
Comment (adellape) [inline installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc:49]: Out of scope for the PR, but just FYI that it's hard for me (mildly red-green colorblind) to tell that these checkmarks were clickable in the rendered version.

**Pull request merged**